### PR TITLE
support prepay charge type and force_delete for resource_mysql_instance

### PR DESCRIPTION
**Pull request by gailwang**
```
make testacc TESTARGS=" -run=TestAccTencentCloudReadonlyMysqlPrepaid"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./... -v -run=TestAccTencentCloudReadonlyMysqlPrepaid -timeout 120m
?       github.com/terraform-providers/terraform-provider-tencentcloud  [no test files]
testing: warning: no tests to run
PASS
ok      github.com/terraform-providers/terraform-provider-tencentcloud/gendoc  (cached) [no tests to run]
=== RUN   TestAccTencentCloudReadonlyMysqlPrepaid
--- PASS: TestAccTencentCloudReadonlyMysqlPrepaid (703.19s)
PASS
ok      github.com/terraform-providers/terraform-provider-tencentcloud/tencentcloud     703.825s
?       github.com/terraform-providers/terraform-provider-tencentcloud/tencentcloud/connectivity        [no test files]
?       github.com/terraform-providers/terraform-provider-tencentcloud/tencentcloud/internal/helper     [no test files]
?       github.com/terraform-providers/terraform-provider-tencentcloud/tencentcloud/ratelimit   [no test files]
```

### CODE REVIEW

File: website/docs/r/mysql_instance.html.markdown [hunk 1]
```diff
@@ -52,15 +52,16 @@ The following arguments are supported:
...
+* `auto_renew_flag` - (Optional) Auto renew flag. NOTES: Only supported prepaid instance.
 * `availability_zone` - (Optional, ForceNew) Indicates which availability zone will be used.
 * `engine_version` - (Optional, ForceNew) The version number of the database engine to use. Supported versions include 5.5/5.6/5.7, and default is 5.7.
 * `first_slave_zone` - (Optional, ForceNew) Zone information about first slave instance.
+* `force_delete` - (Optional) Indicate whether to delete instance directly or not. Default is false. If set true, the instance will be deleted instead of staying recycle bin. Note: only works for `PREPAID` instance. When the main mysql instance set true, this para of the readonly mysql instance will not take effect.
 * `internet_service` - (Optional) Indicates whether to enable the access to an instance from public network: 0 - No, 1 - Yes.
 * `intranet_port` - (Optional) Public access port, rang form 1024 to 65535 and default value is 3306.
 * `parameters` - (Optional) List of parameters to use.
 * `pay_type` - (Optional, ForceNew) Pay type of instance, 0: prepay, 1: postpay. NOTES: Only supported prepay instance.
```
Comment (likexian) [inline website/docs/r/mysql_instance.html.markdown:63]: 这还有个 prepay  可以改掉

File: tencentcloud/resource_tc_mysql_instance_test.go [hunk 2]
```diff
@@ -203,6 +234,39 @@ resource "tencentcloud_mysql_instance" "mysql_master" {
...
+  availability_zone = "ap-guangzhou-3"
+  first_slave_zone  = "ap-guangzhou-3"
+  force_delete = false
+}`
+}
+
+func testAccMysqlMasterInstance_prepaidupdate() string {
+	return `
+resource "tencentcloud_mysql_instance" "prepaid" {
+  pay_type          = 0
```
Comment (oliverpei) [inline tencentcloud/resource_tc_mysql_instance_test.go:256]: 这个字段不是要统一成charge_type吗？

Comment (likexian) [reply tencentcloud/resource_tc_mysql_instance_test.go:256]: 如果要统一，注意要先标成废弃，不要直接修改或删除

File: tencentcloud/resource_tc_mysql_instance.go [hunk 3]
```diff
@@ -71,14 +71,14 @@ func TencentMsyqlBasicInfo() map[string]*schema.Schema {
 			Optional:     true,
```
Comment (brickzzhang) [inline tencentcloud/resource_tc_mysql_instance.go:71]: 是否需要逐步把period改成prepaid_period？

Comment (brickzzhang) [reply tencentcloud/resource_tc_mysql_instance.go:71]: 67行描述中prepay可修改为prepaid

File: tencentcloud/data_source_tc_mysql_instance.go [hunk 4]
```diff
@@ -224,6 +231,11 @@ func dataSourceTencentCloudMysqlInstance() *schema.Resource {
 							Computed:    true,
 							Description: "Pay type of instance, 0: prepay, 1: postpay. NOTES: Only prepay is supported.",
 						},
+						"charge_type": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "Pay type of instance, ``PREPAID``: prepay, `POSTPAID`: postpay. NOTES: Only prepay is supported.",
```
Comment (brickzzhang) [inline tencentcloud/data_source_tc_mysql_instance.go:237]: ``PREPAID``--->`PREPAID`

File: tencentcloud/resource_tc_mysql_instance.go [hunk 5]
```diff
@@ -1159,7 +1205,7 @@ func mysqlUpdateInstancePayByMonth(ctx context.Context, d *schema.ResourceData,
 		d.SetPartial("auto_renew_flag")
 	}
 
-	if d.HasChange("period") {
+	if d.HasChange("period") || d.HasChange("prepaid_period") {
```
Comment (Himer) [inline tencentcloud/resource_tc_mysql_instance.go:1208]: fmt.Errorf("After the initialization of the field[%s] to set does not make sense", "period or prepaid_period")

**Pull request merged**